daemon/daemon_main.go

```diff
@@ -721,6 +726,18 @@ func init() {
 	flags.Bool(option.DisableCNPStatusUpdates, false, "Do not send CNP NodeStatus updates to the Kubernetes api-server (recommended to run with `cnp-node-status-gc=false` in cilium-operator)")
 	option.BindEnv(option.DisableCNPStatusUpdates)
 
+	flags.StringSlice(option.HubbleListenAddresses, []string{}, "List of IP addresses for Hubble server to listen to.")
+	option.BindEnv(option.HubbleListenAddresses)
+
+	flags.Int(option.HubbleFlowBufferSize, 4096, "Maximum number of flows in Hubble's buffer.")
```

Comment:
nit: We should document that this is rounded up to the next power of two.

Reply:
Also, this will be rounded up to 8192 by Hubble. But that's something we have to fix in Hubble itself, not in this PR.

Reply:
The default could be set to `4095` so that it gets rounded to `4096` to minimize this effect for now (at least for the default value).

Reply:
done